### ISSUE TITLE
Subscribe: English error message appears in French page

### ISSUE THREAD

**Issue by XiuJuanXia** [mmiron-HC/WST-RSAMS#106]
URL: https://hcdev.openplus.ca/fr/form/subscribe
User Agent: Chrome
Language: French

Issue Description: In French page, if leave any required field empty and activate the “Soumettre” button, English error message appears.
![image](https://user-images.githubusercontent.com/88776404/136957902-bb11e378-324e-44df-9c29-d1c974901745.png)
![image](https://user-images.githubusercontent.com/88776404/136957997-14b1e60c-7ec3-4e50-95fd-5a311256e723.png)
![image](https://user-images.githubusercontent.com/88776404/136958340-d6cd12d1-a9ce-4a75-b366-c4c4785e8bbe.png)
![image](https://user-images.githubusercontent.com/88776404/136958555-d84398ce-26ac-42c6-b68c-d8648e91c914.png)

Steps to reproduce:

1. Navigate through the forms and leave the forms empty
2. Activate the "Soumettre" button
3. English error message appears.
4. Try to leave three different required fields empty and English error message always appears.

Expected results: In French page, error message should be translated into French.

**Comment by echouin** [comment]
@XiuJuanXia This is related to there being no error messages (similar to #105).﻿

﻿The reason the error message is in english is because it is the browser default message. The browser will use whatever default language is being used by the browser/OS.

Suggest Close?

**Comment by XiuJuanXia** [comment]
@echouin, tested in Chrome and Firefox, the three fields that have red asterisk * at the end are read "required" by screen readers. And also if checking the source code,  required="required" for the input fields. In this case, if we leave the three fields empty and submit the form, proper error message should be triggered in proper language.

**Comment by echouin** [comment]
From OP: the reason for the alerts in the wrong language has to do with the client browser/OS config, not Drupal. There is a related issue being worked on here #105 , suggest that this one be closed in favor of 105 (which should solve both issues).

**Comment by mmiron-HC** [comment]
@echouin we do not close issues until the item has passed testing.

**Comment by echouin** [comment]
@mmiron-HC This is a Drupal WxT issue. OP unable to fix at this time.

**Comment by davidgotlieb** [comment]
Should be resolved with the fix that was implemented for 105

**Comment by hydegeoff** [comment]
this issues were resolved. Issue closed.
